tools/scaffolding/scaffolder.go

```diff
@@ -79,6 +79,23 @@ func determineGoPath() string {
 	return strings.TrimSpace(string(goPathOut))
 }
 
+func determineYarnPath() string {
+	path, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	yarnScript := filepath.Join(path, "..", "..", "build", "bin", "yarn.sh")
+	scriptPath, err := exec.LookPath(yarnScript)
+	if err != nil {
+		path, err := exec.LookPath("yarn")
+		if err != nil {
+			log.Fatal("could not find yarn executable to use")
+		}
+		return strings.TrimSpace(path)
```

Comment:
what's with the TrimSpace?

Reply:
I was following the pattern from `determineGoPath` which I assume did it to be safe

Reply:
that just comes from the fact that it's executing a command that returns with a newline. i don't think any of your commands do that.